Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -1052,7 +1052,7 @@ def audit_version
     end
 
     def audit_download_strategy
-      if url =~ %r{^(cvs|bzr|hg|fossil)://} || url =~ %r{^(svn)\+http://}
+      if url =~ %r{^(svn)\+http://}
```

Comment:
I hesitated to migrate this one since I'm not sure about `svn+http://`. I'm no svn expert but I've never seen this scheme and can't find any documentation about this case. I know there is `svn+ssh://` for example. See also https://en.wikipedia.org/wiki/Apache_Subversion

Any input appreciated :)

Reply:
Everything on this list was fake schemes, used historically to indicate to `brew` which download strategy to use. This audit was introduced to deprecate them: 0344d243e4b7279490d9be806ee440fcdcb102b3

Reply:
Thanks for the historical context @Bo98! 
Migrated this one too therefore 👍.